iamproperty/iamproperty.github.io
# Utility classes

## Summary of Changes:
1. Documentation utility classes
2. Remove some unwanted ones

## Rationale
Document which utility classes are used and removed from the bootstrap utility api.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /foundations/utilities. 

## Ticket(s)
DS-29

## Checklist

## Notes
-
